public/app/routes/routes.tsx

```diff
@@ -32,6 +32,10 @@ export function getAppRoutes(): RouteDescriptor[] {
           path: '/apps',
           component: () => <NavLandingPage navId="apps" />,
         },
+        {
+          path: '/alerts-and-incidents',
+          component: () => <NavLandingPage navId="alerts-and-incidents" />,
```

Comment:
Should this be a dynamic import?

Reply:
the NavLandingPage is not lazyily specified in other places so just used the same